source/includes/_local_development.md

```diff
@@ -0,0 +1,40 @@
+# Local Development
+
+Ready to get started developing Origin.js and the Demo DApp? You will need two terminal windows, one for origin.js and one for the DApp.
+
+Origin.js:
+
+<pre style="margin-right: 50%; float:none; width: inherit; margin-bottom: 10px">
+    mkdir -p origin
+    cd origin
+    git clone https://github.com/OriginProtocol/origin-js.git
+    cd origin-js
+    git checkout develop
+    npm install
+    npm link
+    npm start
+</pre>
+
+And in another window, let's get the Demo DApp going:
+
+<pre style="margin-right: 50%; float:none; width: inherit; margin-bottom: 10px">
+    mkdir -p origin
+    cd origin
+    git clone https://github.com/OriginProtocol/demo-dapp.git
+    cd demo-dapp
+    git checkout develop
+    npm install
+    npm link origin
+    npm start
+</pre>
+
+Your local repos have been setup to use the develop branche, since that's where all our new work starts from. You have also linked the Demo Dapp to use your local copy of the origin.js library, so that your changes to origin.js are reflected immediately.
```

Comment:
I assume `branche` is a typo? Unless it's a alternate spelling I'm not aware of. 😄

Reply:
I think he probably intended the plural form there. I've fixed it and changed the commands to match the READMEs.